src/executor/senderManager.ts

```diff
@@ -251,6 +252,12 @@ export class SenderManager {
             { executor: wallet.address },
             "pushed wallet to sender manager"
         )
+        const processingTime = this.walletProcessingTime.get(wallet.address)
+        if (processingTime) {
```

Comment:
do we really need this check now? should be always non-nullable

Reply:
TS screams otherwise :(